src/main/java/data/JsonConverter.java

```diff
@@ -142,18 +146,22 @@ public boolean configExists(String vendorID) {
     try {
       String config = getConfig(vendorID);
       Vendor vendor = new Vendor(config, vendorID);
-      ArrayList<Account> accounts = vendor.getAccounts();
       ArrayList<String> accountIds = new ArrayList<String>();
+      vendor.getAccounts().forEach(account -> accountIds.add(account.getAccountID()));
```

Comment:
fancy

Reply:
😎

Reply:
This looks great. I think you could do the same thing with .stream().map(...).collect(), but I'm happy with this.